imxrt-hal/src/iomuxc/pin_config.rs

```diff
@@ -0,0 +1,218 @@
+/// Pin configuration provides types, macros, and methods for
+/// setting a pin configuration such as pull resistors, speed, and
+/// open drain options.
+
+/// Slew rate mask, offset, and possible values
+pub mod slew_rate {
+    use crate::ral;
+
+    pub const MASK: u32 = ral::iomuxc::SW_PAD_CTL_PAD_GPIO_EMC_00::DSE::mask;
+    pub const OFFSET: u32 = ral::iomuxc::SW_PAD_CTL_PAD_GPIO_EMC_00::DSE::offset;
```

Comment:
Looks like we're using `DSE` here. Should this be `SRE` for slew rate?

Comment also applies to the `MASK` definition.

Reply:
Yes, thank you